euler/ss-problem-84/p84.cpp

```diff
@@ -0,0 +1,271 @@
+#include <algorithm>
+#include <iostream>
+#include <ostream>
+#include <vector>
+
+#include <Eigen/Dense>
+
+using Eigen::MatrixXd;
+
+/*
+ Compute the probability distribution of the squares of a Monopoly board as a markov chain stable distribution.
+ */
+
+// C++ template to print vector container elements
+template <typename T>
+std::ostream& operator<<(std::ostream& os, const std::vector<T>& v)
+{
+    os << std::endl;
+    for (int i = 0; i < v.size(); ++i) {
+        os << v[i];
+        if (i != v.size() - 1)
+            os << ", ";
+    }
+    os << std::endl;
+    return os;
+}
+
+std::vector<int> polynomial_multiply(std::vector<int> p, std::vector<int> q){
```

Comment:
This kind of basic math operation probably belongs in a shared library so that other Euler problems can use it